docs/java-rest/low-level/usage.asciidoc

```diff
@@ -206,6 +206,12 @@ include-tagged::{doc-tests}/RestClientDocumentation.java[rest-client-init-reques
 https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/client/config/RequestConfig.Builder.html[`org.apache.http.client.config.RequestConfig.Builder`]
  allows to set)
 
+[[java-rest-low-usage-initialization-proxy]]
+==== Using a proxy
+
+When running Elasticsearch behind a proxy, you will need to configure the
+low level REST client this way:
+
```

Comment:
Shall we move this below the example that shows the encrypted communication configuration? (below line 222)

I think the encrypted communication example still stands as an example on its own and we can showcase the proxy path configuration a standalone paragraph and code snippet